frame/fast-unstake/src/lib.rs

```diff
@@ -113,10 +113,12 @@ pub mod pallet {
 			+ IsType<<Self as frame_system::Config>::RuntimeEvent>
 			+ TryInto<Event<Self>>;
 
-		/// The amount of balance slashed per each era that was wastefully checked.
-		///
-		/// A reasonable value could be `runtime_weight_to_fee(weight_per_era_check)`.
-		type SlashPerEra: Get<BalanceOf<Self>>;
+		/// The currency used for deposits.
+		type DepositCurrency: ReservableCurrency<Self::AccountId, Balance = Self::CurrencyBalance>;
```

Comment:
```suggestion
		type Currency: ReservableCurrency<Self::AccountId, Balance = Self::CurrencyBalance>;
```

Reply:
This is already defined in pallet_staking which inherited by this one.

Reply:
You can disambiguate by stating `<T as Config>::Currency` instead of `T::Currency`. Do whichever makes more sense to you, no strong opinion.